packages/components/inputs/select-input/README.md

```diff
@@ -73,6 +73,7 @@ export default Example;
 | `isAutofocussed`           | `boolean`                                                                                                |          |             | Focus the control when it is mounted                                                                                                                                                                                                                                                                                                                                                     |
 | `backspaceRemovesValue`    | `ReactSelectProps['backspaceRemovesValue']`                                                              |          |             | Remove the currently focused option when the user presses backspace&#xA;<br>&#xA;[Props from React select was used](https://react-select.com/props)                                                                                                                                                                                                                                      |
 | `components`               | `ReactSelectProps['components']`                                                                         |          |             | Map of components to overwrite the default ones, see what components you can override&#xA;<br>&#xA;[Props from React select was used](https://react-select.com/props)                                                                                                                                                                                                                    |
+| `isCondensed`              | `boolean`                                                                                                |          |             | Is the component is displayed in a condensed layout                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
I think we need to regenerate the Readme in order to get the latest JSDoc of this property

Reply:
Readme updated here: cc7989be06c2d337ef6f7408629d9b0909a912a5